lib/Launcher.js

```diff
@@ -80,6 +80,8 @@ class Launcher {
           '--mute-audio'
       );
     }
+    if (options.headless === false && options.devtools === true)
+      chromeArguments.push('--auto-open-devtools-for-tabs');
```

Comment:
@vsemozhetbyt we discussed and folks are comfortable with this. (including me. :)

 can you also set `headless:false` automatically if `devtools` is true?

Reply:
Is it OK?